### PR TITLE
DT-1440 Single Key dates deleted on all active custodial events

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: src/test/java/uk/gov/justice/digital/delius/util/EntityHelper.java [hunk 1]
```diff
@@ -498,17 +498,6 @@ private static Disposal aDisposal(final Long eventId) {
                 .build();
     }
 
-    public static Disposal aCommunityDisposal(final Long eventId) {
```
Comment (andymarke) [inline src/test/java/uk/gov/justice/digital/delius/util/EntityHelper.java:501]: not used

File: src/test/java/uk/gov/justice/digital/delius/util/EntityHelper.java [hunk 2]
```diff
@@ -224,7 +224,7 @@ private static Custody aCustody(final Disposal disposal, final List<KeyDate> key
                 .builder()
                 .disposal(disposal)
                 .custodyId(9999L)
-                .keyDates(keyDates)
+                .keyDates(new ArrayList<>(keyDates))
```
Comment (andymarke) [inline src/test/java/uk/gov/justice/digital/delius/util/EntityHelper.java:227]: dates need to be in a mutable list since they are removed

**Review by dnyanesh-sonwane — APPROVED** [review]
Looks good

**Review by chris-faulkner-mojdt — APPROVED** [review]
LGTM

**Pull request merged**